pull-test.sh

```diff
@@ -26,7 +26,7 @@ CSI_RELEASE_TOOLS_DIR="$(pwd)"
 # Update the other repo.
 cd "$PULL_TEST_REPO_DIR"
 git reset --hard # Shouldn't be necessary, but somehow is to avoid "fatal: working tree has modifications.  Cannot add." (https://stackoverflow.com/questions/3623351/git-subtree-pull-says-that-the-working-tree-has-modifications-but-git-status-sa)
-git subtree pull --squash --prefix=release-tools "$CSI_RELEASE_TOOLS_DIR" master
+GIT_NO_LAZY_FETCH=0 git subtree pull --squash --prefix=release-tools "$CSI_RELEASE_TOOLS_DIR" master
```

Comment:
Please add a comment why we need GIT_NO_LAZY_FETCH and perhaps put it to  a standalone line

Reply:
Rebased onto #264